src/Main.cpp

```diff
@@ -56,9 +55,6 @@ static const struct OptionTemplate
     {'D', "debug", OptionType::kBool, offsetof(DriverOptions, m_DebugMessages), "Enable debug messages"},
     {'S', "debug-signing", OptionType::kBool, offsetof(DriverOptions, m_DebugSigning), "Generate an extensive log of signature generation"},
     {'e', "just-print-leafinput-signature", OptionType::kString, offsetof(DriverOptions, m_JustPrintLeafInputSignature), "Print to the specified file the leaf input signature ingredients of the requested node"},
-    {'r', "throttle", OptionType::kBool, offsetof(DriverOptions, m_ThrottleOnHumanActivity), "Throttles down amount of simultaneous jobs when mouse or keyboard activity has been detected."},
-    {'\0', "throttle-time", OptionType::kInt, offsetof(DriverOptions, m_ThrottleInactivityPeriod), "Amount of inactive time after which we stop throttling. (if throttling behaviour is enabled)"},
-    {'\0', "throttle-threads-amount", OptionType::kInt, offsetof(DriverOptions, m_ThrottledThreadsAmount), "Amount of threads used in throttled mode"},
```

Comment:
Just checking- do we use these flags on any build infra? Alternatively we could leave the flags but make them no-ops.

Reply:
we do not, this feature never really worked well.